src/types/my-offers/interfaces/myOffers.interfaces.ts

```diff
@@ -0,0 +1,8 @@
+import { RequestParams } from '~/types/services/services.index'
+import { MyCooperationsFilters, UserResponse } from '~/types'
+
+export interface GetMyOffersParams
+  extends Partial<MyCooperationsFilters>,
+    RequestParams {
+  id: UserResponse['_id']
```

Comment:
if this is an offer, pick field id from offer

Reply:
fixed